src/ol_infrastructure/infrastructure/applications/dagster/Pulumi.applications.dagster.Production.yaml

```diff
@@ -2,3 +2,12 @@ secretsprovider: awskms://alias/PulumiSecrets
 encryptedkey: AQICAHionUR8LBW1ALuVC0rCH3AE2oQIfGMCx3XmpDH9HjM2LQEGccCx6w+aU1z9VNPzXzwIAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMkTCVUUXv23ZgiFtbAgEQgDusl6M2Rs0caUtV7jakSKAqsXTqm8f7M0UuEu0hjZ5KdbCp8HUDBShST/yKEpt0ExGXlODrlmL3gGN6fg==
 config:
   aws:region: us-east-1
+  dagster:instance_type: m5a.large
+  dagster:domain: pipelines.odl.mit.edu
+  dagster:db_password:
```

Comment:
Is this supposed to be `saltstack:api_password` cause that's what's in dagster.QA

Reply:
No, it's for the db password. I haven't added the salt api password because we haven't deployed those changes to production yet.